Dockerfile.dev

```diff
@@ -1,6 +1,6 @@
 FROM ubuntu:xenial
 
 RUN apt-get update && apt-get -y install \
-    curl unzip git build-essential ca-certificates
+    curl unzip git build-essential ca-certificates libssl-dev
```

Comment:
Do you mean if the CI nodes includes the library? The `libssl-dev` library is only used by the TPM simulator (in unit tests). Since unit tests passed, I think the library is pre-installed.

Reply:
Yes, I was referring to that. Good point, if the tests are passing in the PR then there should not be a problem.